[2024-06-11T04:13:50.212+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-11T04:13:50.232+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_word.process_file manual__2024-06-11T04:13:48.727513+00:00 [queued]>
[2024-06-11T04:13:50.239+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_word.process_file manual__2024-06-11T04:13:48.727513+00:00 [queued]>
[2024-06-11T04:13:50.240+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-11T04:13:50.255+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_file> on 2024-06-11 04:13:48.727513+00:00
[2024-06-11T04:13:50.262+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-11T04:13:50.263+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_word', 'process_file', 'manual__2024-06-11T04:13:48.727513+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/simple_hello_world.py', '--cfg-path', '/tmp/tmpwlw3wjvd']
[2024-06-11T04:13:50.264+0000] {standard_task_runner.py:63} INFO - Started process 1125 to run task
[2024-06-11T04:13:50.264+0000] {standard_task_runner.py:91} INFO - Job 75: Subtask process_file
[2024-06-11T04:13:50.333+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_word.process_file manual__2024-06-11T04:13:48.727513+00:00 [running]> on host 678b3b027d57
[2024-06-11T04:13:50.402+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Anudeep' AIRFLOW_CTX_DAG_ID='hello_word' AIRFLOW_CTX_TASK_ID='process_file' AIRFLOW_CTX_EXECUTION_DATE='2024-06-11T04:13:48.727513+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-11T04:13:48.727513+00:00'
[2024-06-11T04:13:50.404+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-11T04:13:50.427+0000] {logging_mixin.py:188} INFO - None ['sample1.json', 'sample2.json']
[2024-06-11T04:13:50.427+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-11T04:13:50.429+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/simple_hello_world.py", line 46, in process_file
    with open(os.path.join(path, filename)) as file:
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 76, in join
TypeError: expected str, bytes or os.PathLike object, not NoneType
[2024-06-11T04:13:50.442+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=hello_word, task_id=process_file, run_id=manual__2024-06-11T04:13:48.727513+00:00, execution_date=20240611T041348, start_date=20240611T041350, end_date=20240611T041350
[2024-06-11T04:13:50.455+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 75 for task process_file (expected str, bytes or os.PathLike object, not NoneType; 1125)
[2024-06-11T04:13:50.479+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-11T04:13:50.515+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-11T04:13:50.601+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
